src/reference/asciidoc/images/epub-cover.svg

```diff
@@ -6,7 +6,11 @@
   <text x="250" y="192" fill="Black" font-size="72">Reference Guide</text>
   <text x="250" y="420" fill="Black" font-size="36">Mark Fisher, Marius Bogoevici,</text>
   <text x="250" y="466" fill="Black" font-size="36">Iwein Fuld, Jonas Partner,</text>
+<<<<<<< HEAD
```

Comment:
oops.   Need to remove the merge comments ;-)

Reply:
Thanks, Glenn. I fixed it. The aggravating thing is that I searched for HEAD with grep and didn't find it. Sigh.